components/collections/CollectionDetails.tsx

```diff
@@ -44,7 +44,8 @@ export const CollectionDetails: FC<Props> = ({
   const chainCurrency = useChainCurrency()
 
   const descriptionRef = useRef(null as any)
-  const contractKind = collection?.contractKind?.toUpperCase()
+
+  const contractKind = `${collection?.contractKind?.toUpperCase()}${collection?.securityConfig && 'C'}`
```

Comment:
It looks like when `includeSecurityConfigs` is set to true in the collections request, the `securityConfig` object is always returned. This is causing every collection to display as an 'ERC721C'. I think we need to establish another way to verify this